static/usage/v7/select/customization/custom-toggle-icons/vue.md

```diff
@@ -0,0 +1,28 @@
+```html
+<template>
+  <ion-list>
+    <ion-item>
+      <ion-select
+        interface="popover"
+        toggle-icon="add"
+        toggle-icon-when-open="remove"
```

Comment:
Ionic Vue needs the icons imported from `ionicons/icons` and then passed to the props

```typescript
import { add, remove } from 'ionicons/icons';

...

:toggle-icon="add"
```